builtin/game/register.lua

```diff
@@ -514,11 +514,18 @@ local function make_registration_wrap(reg_fn_name, clear_fn_name)
 end
 
 local function make_wrap_deregistration(reg_fn, clear_fn, list)
-	local unregister = function (unregistered_key)
+	local unregister = function (key)
+		if type(key) ~= "string" then
+			error("Argument passed to " .. debug.getinfo(1, "n").name ..
```

Comment:
Huh? Why not `key is not a string`?

Reply:
It should be a string right? `core.unregister_biome(biome_name)`

Reply:
Ah, I think I misunderstood - did you mean to simplify the error message?

Reply:
There will be a stacktrace on error, so I don't think the verboseness is needed.

Reply:
Good point. Done.